zerver/tests/test_hooks.py

```diff
@@ -14,10 +15,10 @@ class WebhookTestCase(AuthedTestCase):
     If you create your url in uncommon way you can override build_webhook_url method
     In case that you need modify body or create it without using fixture you can also override get_body method
     """
-    STREAM_NAME = None
+    STREAM_NAME = None # type: text_type
```

Comment:
Shouldn't this be `Optional[text_type]`?

I'm not sure whether mypy automatically types a variable as `Optional[text_type]` instead of `text_type` if it sees a `None` being assigned to it.